source/Generic/LibraryExporter/ViewModels/LibraryExporterViewModel.cs

```diff
@@ -99,6 +99,7 @@ private List<string> GetColumnsList()
             if (Settings.Settings.ExportSettings.Modified) { columnNames.Add(GetLocalizationString("LOCDateModifiedLabel")); }
             if (Settings.Settings.ExportSettings.RecentActivity) { columnNames.Add(GetLocalizationString("LOCRecentActivityLabel")); }
             if (Settings.Settings.ExportSettings.Roms) { columnNames.Add("Roms"); }
+            if (Settings.Settings.ExportSettings.CompletionStatus) { columnNames.Add("CompletionStatus"); }
```

Comment:
Should also use Playnite localization string

Reply:
Same here I will use LOCCompletionStatus.